internal/serve/httphandler/health_handler_test.go

```diff
@@ -1,38 +1,112 @@
 package httphandler
 
 import (
+	"errors"
 	"net/http"
 	"net/http/httptest"
 	"testing"
 
+	"github.com/stellar/stellar-disbursement-platform-backend/db"
+	"github.com/stellar/stellar-disbursement-platform-backend/db/dbtest"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/events"
+	"github.com/stretchr/testify/mock"
+
 	"github.com/go-chi/chi/v5"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 // test HealthHandler:
 func TestHealthHandler(t *testing.T) {
-	// setup
+	// create database connection pool
+	dbt := dbtest.OpenWithoutMigrations(t)
+	defer dbt.Close()
```

Comment:
💡  You can test the database with a failed status by creating a new dbConnectionPool and closing it before the ping is called.

Reply:
great idea ! Added a few more tests that use that.

Addressed in d854b5f9e9ef197be9157b3b07c29e1a9f852fc7